ubuntu_package_changelog/__init__.py

```diff
@@ -48,6 +69,16 @@ def _parser():
                         type=_args_validate_ppa_name)
     parser.add_argument('--entries', help='number of changelog entries to get from the '
                         'changelog. 0 mean all. Default: %(default)s', type=int, default=1)
+    parser.add_argument('--binary-package',
+                        help='Set this option if you wish to get the changelog for the '
+                        'source package of the specified binary package name. '
+                        'Default: %(default)s',
+                        action='store_true', default=False)
+    parser.add_argument('--binary-package-architecture',
```

Comment:
would it be possible to drop that flag and just use whatever architecture a binary package has? The source package and changelog is not different between the architectures.